src/frontend/src/lib/components/tokens/TokenContext.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import { setContext } from 'svelte';
+	import { initTokenStore } from '$lib/stores/token.store';
+
+	const tokenContextKey = {};
```

Comment:
Shouldn't this component takes a `token` as property and use it to create the context?

That way the context is never undefined or empty.

Reply:
You are right! Fixed